app/src/main/webapp/predictor.jsp

```diff
@@ -14,21 +14,109 @@
+                            <option value="${country}"><c:out value="${country}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+                <td>
+                    <label for="country2">Country: </label>
+                    <select id="country2" required>
+                        <c:forEach var="country" items="${listCountries.rows}">
+                            <option value="${country}"><c:out value="${country}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <label for="league1">League: </label>
+                    <select id="league1" required disabled>
+                        <c:forEach var="league" items="${listLeagues.rows}">
+                            <!-- league.get(0) is the name of the league, and league.get(1) is the country of the league -->
+                            <option class=="${fn:replace(${league.get(1)}, ' ', '_')}" value="${league.get(0)}"><c:out value="${league.get(0)}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+                <td>
+                    <label for="league2">League: </label>
+                    <select id="league2" required disabled>
+                        <c:forEach var="league" items="${listLeagues.rows}">
+                            <!-- league.get(0) is the name of the league, and league.get(1) is the country of the league -->
+                            <option class=="${fn:replace(${league.get(1)}, ' ', '_')}" value="${league.get(0)}"><c:out value="${league.get(0)}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <label for="team1">Team: </label>
+                    <select id="team1" required disabled>
+                        <c:forEach var="team" items="${listTeams.rows}">
+                            <option class=="${fn:replace(${team.get(1)}, ' ', '_')}" value="${team.get(0)}"><c:out value="${team.get(0)}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+                <td>
+                    <label for="team2">Team: </label>
+                    <select id="team2" required disabled>
+                        <c:forEach var="team" items="${listTeams.rows}">
+                            <option class=="${fn:replace(${team.get(1)}, ' ', '_')}" value="${team.get(0)}"><c:out value="${team.get(0)}" /></option>
+                        </c:forEach>
+                    </select>
+                </td>
+            </tr>
+        </table>
+    </div>
     <div>
         <button id="predict-btn" type="submit" name="predict-btn" class="btn btn-primary"
-            disabled>Predict</button>
+            onclick"">Predict</button>
```

Comment:
this works ?

Reply:
No, I never assigned it to a function because I didn't know how to make the function